### PR TITLE
PMM-10653-Vacuum-monitoring-Experimental-dashboard

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

File: docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+This dashbaord provides timely insights into the autovacuum process in PostgreSQL.
+
+This dashboard contains the following:
+
+- Dead tuples - Identifies the number of dead rows in each table even though the rows are physically removed from the table.
+
+- Last time vacuum ran - Tracks the last time a vacuum or autovacuum process successfully ran on each of your tables.
+
```
Comment (dygksy) [inline docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md:17]: I couldnt find the last 3 metrics/data on the dashboard. Can you check it again?

**Review by dygksy — CHANGES_REQUESTED** [review]

File: docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md:17]: @JiriCtvrtka Could you please confirm if we capture these metrics?

Comment (JiriCtvrtka) [reply docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md:17]: Verified with Anton and we have these metrics even after last required changes from DGB.

Running instance with latest changes:

http://3.140.254.197/graph/d/postgres_vacuum_monitoring/postgresql-vacuum-monitoring?orgId=1&refresh=1m

Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md:17]: Thanks @JiriCtvrtka for confirming.

**Review by JiriCtvrtka — APPROVED** [review]

**Review by anton-bystrov — APPROVED** [review]

**Review by dygksy — APPROVED** [review]

**Review by dygksy — APPROVED** [review]

**Review by nethalo — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**